BusinessLogic/Abstractions/IUserFactory.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace PrivateClinicsWebNet.BusinessLogic.Abstractions
+{
+    public interface IUserFactory
+    {
+        IdentityUser GetUser(string email, string identityRole);
```

Comment:
You don't get it from some place, but u probably create it. Otherwise it's not a factory